javagen/src/main/java/com/azure/autorest/util/ModelExampleUtil.java

```diff
@@ -230,6 +236,101 @@ public static Object getChildObjectValue(List<String> jsonPropertyNames, Object
+                node = new LiteralNode(ClassType.Context, "").setLiteralsValue("");
+            } else {
+                node = new LiteralNode(methodParameter.getClientMethodParameter().getClientType(), null);
+            }
+        } else {
+            Object exampleValue = methodParameter.getClientMethodParameter().getRequestParameterLocation() == RequestParameterLocation.QUERY
+                    ? parameterValue.getUnescapedQueryValue()
+                    : parameterValue.getObjectValue();
+            node = parseNodeFromMethodParameter(methodParameter, exampleValue);
+        }
+        return node;
+    }
+
+    /**
+     * Find parameter example value from proxy method example by serialized parameter name.
+     * @param example proxy method example
+     * @param serializedName parameter serialized name
+     * @return example value for this parameter
+     */
+    public static ProxyMethodExample.ParameterValue findParameter(ProxyMethodExample example, String serializedName) {
+        return example.getParameters().entrySet()
+                .stream().filter(p -> p.getKey().equalsIgnoreCase(serializedName))
+                .map(Map.Entry::getValue)
+                .findFirst().orElse(null);
+    }
+
+    private static ExampleNode parseNodeFromMethodParameter(MethodParameter methodParameter, Object objectValue) {
+        IType type = methodParameter.getClientMethodParameter().getClientType();
+        IType wireType = methodParameter.getClientMethodParameter().getWireType();
+        if (methodParameter.getProxyMethodParameter().getCollectionFormat() != null && type instanceof ListType && objectValue instanceof String) {
+            // handle parameter style
+
+            IType elementType = ((ListType) type).getElementType();
+            ListNode listNode = new ListNode(elementType, objectValue);
+            String value = (String) objectValue;
+
+            CollectionFormat collectionFormat = methodParameter.getProxyMethodParameter().getCollectionFormat();
+            List<String> elements;
+            switch (collectionFormat) {
+                case CSV:
+                    elements = Arrays.asList(value.split(Pattern.quote(","), -1));
+                    break;
+                case SSV:
+                    elements = Arrays.asList(value.split(Pattern.quote(" "), -1));
+                    break;
+                case PIPES:
+                    elements = Arrays.asList(value.split(Pattern.quote("|"), -1));
+                    break;
+                case TSV:
+                    elements = Arrays.asList(value.split(Pattern.quote("\t"), -1));
+                    break;
+                default:
+                    // TODO (weidxu): CollectionFormat.MULTI
+                    elements = Arrays.asList(value.split(Pattern.quote(","), -1));
+                    LOGGER.error("Parameter style '{}' is not supported, fallback to CSV", collectionFormat);
+                    break;
```

Comment:
Emm, why I say this?

OK, I assume I am not sure how service write the MULTI in example json. So current code assume it is "," separated values.